react-form-controls/src/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+export default class RegistrationForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      passValue: '',
+      nameValue: ''
+    };
+    this.handleUsernameChange = this.handleUsernameChange.bind(this);
+    this.handlePasswordChange = this.handlePasswordChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleUsernameChange(event) {
+    this.setState({ nameValue: event.target.value });
+  }
+
+  handlePasswordChange(event) {
+    this.setState({ passValue: event.target.value });
+  }
+
+  handleSubmit(event) {
+    console.log('New user signed up:' + this.state.nameValue + ' ' + this.state.passValue);
+    event.preventDefault();
+    this.setState({ nameValue: '', passValue: '' });
+  }
+
+  render() {
+    return (
+    <form>
+      <label>
+        Username:
+        <input type="text" value={this.state.nameValue} onChange={this.handleUsernameChange} />
+      </label>
+      <label>
+        Password:
+          <input type="password" value={this.state.passValue} onChange={this.handlePasswordChange} />
+      </label>
+      <button onClick={this.handleSubmit}>Submit</button>
```

Comment:
`handleSubmit` should be on the form and not the button

Reply:
done